src/app/mytools/mytools.routing.ts

```diff
@@ -21,7 +21,6 @@ import { MyToolsComponent } from './mytools.component';
 const MYTOOLS_ROUTES: Routes = [
   {
     path: '', component: MyToolsComponent, children: [
-      { path: '', component: MyToolComponent},
```

Comment:
Any explanation why this stops a 2nd component from being created?  "The router uses a first-match wins strategy when matching routes, so more specific routes should be placed above less specific routes."

Reply:
I can look into it.

Reply:
I think it might be due to initially loading the component with no entry selected (so url matches ''), and then it selects an entry and updates the url (so the url matches '**')